sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/models/ContainersGetPropertiesResponse.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Contains all response data for the getProperties operation.
  */
-public final class ContainersGetPropertiesResponse extends ResponseBase<ContainerGetPropertiesHeaders, Void> {
+public final class ContainersGetPropertiesResponse extends ResponseBase<BlobContainerGetPropertiesHeaders, Void> {
```

Comment:
Codegen class, these changes will be reverted in the next generation.

Reply:
Reverted